product_docs/docs/postgres_distributed_for_kubernetes/1/managed.mdx

```diff
@@ -1,12 +1,12 @@
 ---
-title: 'Managed Configuration'
+title: 'Managed configuration'
 originalFilePath: 'src/managed.md'
 ---
 
-The PGD operator allows configuring the `managed` section of a PG4K cluster. The `spec.cnp.managed` stanza
-is used for configuring the supported managed roles within the cluster.
+The PGD operator allows configuring the `managed` section of a PGD4K cluster. The `spec.cnp.managed` stanza
```

Comment:
Please check this edit. Was it meant to say PG4K or PGD4K, which is what this doc is about?

Reply:
It was meant for PG4K, without the D.
The confusing bit here: PGD4K ... let me call it PG4K-PGD for better contrast ... leverages the PG4K operator under the covers. The thing the PG4K operator manages is "clusters". In PG4K-PGD we tend to manage PGD Groups.
Hope that helps.

Reply:
The `spec.cnp.managed` stanza allows us to pass some PG4K parameters along to the PG4K "level".

Reply:
Let me know if any other clarification is in order here for the doc.